### ISSUE TITLE
[Feature request] modifying keys that are being passed to other applications

### ISSUE THREAD

**Issue by zmeyka3310** [hyprwm/Hyprland#8130]
### Description

My mainmod key is alt, and a lot of keybinds in apps use alt as the in-app mod key, so my super has no binds bound to it in hyprland nor in apps. i would like that when i press super, the apps recieve an alt, so that i could use the in-app keybinds by pressing super (and it wouldnt interfere with hyprland), while retaining the functionality to use alt as an alt key. i tried to do this with `bind = SUPER, , sendshortcut, ALT, , ^(.*)$`, but apparently it doesnt work as i want it to. (to clarify, when a key is being held, it will hold the other key too)

tldr when pressing key A hyprland registers key A and passes it as key B to the focused window for the entire duration that key A is being held. the held key should be precieved as key B only by the app, but as key A only for hyprland (or for both)

**Comment by MahouShoujoMivutilde** [comment]
Idk how to do it in a general way, but for your specific use case you can

```ini
input {
    kb_options = altwin:swap_lalt_lwin
}
```

(from man xkeyboard-config)

**Comment by zmeyka3310** [comment]
> Idk how to do it in a general way, but for your specific use case you can
> 
> ```ini
> input {
>     kb_options = altwin:swap_lalt_lwin
> }
> ```
> 
> (from man xkeyboard-config)

the issue is that a swap is not enough. i want Alt to behave like alt globally and in apps, but super to behave as alt only in apps

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
